README.adoc

```diff
@@ -1,6 +1,209 @@
+// Header of this document:
+
 = T2 - Test & Troubleshooting Platform
+:toc:
+:toc-placement: preamble
+:toclevels: 2
+:showtitle:
+:base-repo: https://github.com/stackabletech/t2
+
+// Need some preamble to get TOC:
+{empty}
 
 image:https://github.com/stackabletech/t2/workflows/Build%20and%20Test/badge.svg[link="https://github.com/stackabletech/t2/actions"]
 
-This platform helps us by offering tools for (integration) tests and troubleshooting (support).
+== Motivation
+
+There are two situations in which we need to launch fully-fledged Stackable clusters:
+
+* in our CI/CD pipelines' integration *tests*
+* when we want to help customers with issues by reproducing their environments (*troubleshooting*)
+
+== The two ways to create clusters with T2
+
+A running T2 instance provides a restful API which provides you with two ways to create a cluster:
+
+* The cluster can be launched by T2 inside a connected cloud infrastructure.
```

Comment:
nit: maybe "configured" instead of "connected"

Reply:
I agree that "connected" does not sound right. I omitted the adjective entirely though.